.yarnrc.yml

```diff
@@ -1,5 +1,11 @@
+enableGlobalCache: true
```

Comment:
This defaults to `true`, so should already be on.

Reply:
default unless some other settings are configured. this removes the local (in repo) cache that creates the tar balls in Git. global means the cache is in the home dir which is what we need to clean that up.